db-service/test/cqn2sql/testModel.cds

```diff
@@ -13,6 +13,13 @@ entity Foo2 {
   virtual something : String(11);
 }
 
+entity !["Foo2Quoted"] {
+  key !["ID"]: Integer;
+  !["name"]: String;
+  !["a"]: Integer;
```

Comment:
can you rename/add one of the columns to a reserved keyword as in the bug report?

Reply:
The root cause was not the reserved key word. As this was correctly being detected. It was just the problem `quote` was being called twice.